packages/manager/apps/hub/package.json

```diff
@@ -56,7 +54,8 @@
     "oclazyload": "^1.1.0",
     "ovh-api-services": "^9.41.0",
     "ovh-ui-angular": "^3.15.1",
-    "ovh-ui-kit": "^2.40.0"
+    "ovh-ui-kit": "^2.40.0",
+    "whatwg-fetch": "^3.0.0"
```

Comment:
As  `bootstrapApplication` from `@ovh-ux/manager-core` use `fetch` we need a polyfill.